swagger_server/utils/db_utils.py

```diff
@@ -1,64 +1,51 @@
 import logging
 import os
 
-import _pickle as pickle
-import dataset
+import pymongo
 
+DB_NAME = os.environ.get("DB_NAME")
 DB_CONFIG_TABLE_NAME = os.environ.get("DB_CONFIG_TABLE_NAME")
+MONGODB_CONNSTRING = os.environ.get("MONGODB_CONNSTRING")
 
 
 class DbUtils(object):
     def __init__(self):
-        # self.table_name
-        self.config_table = DB_CONFIG_TABLE_NAME
+        self.db_name = DB_NAME
+        self.config_table_name = DB_CONFIG_TABLE_NAME
+        self.mongo_client = pymongo.MongoClient(MONGODB_CONNSTRING)
         self.logger = logging.getLogger(__name__)
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
What happens when the environment variables are not present at all? Should we do some error handling here, or elsewhere?

Reply:
We'll move the environment variables from docker-compose to manifest soon, will handle this in a separate PR. This PR's scope is mainly about db.